api-model/pom.xml

```diff
@@ -50,6 +50,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
+
```

Comment:
I don't think its justified to add this dependency just for the toString helper methods.

Reply:
Well, we have this dependency already. So I don't think is makes a difference in the final application.

Reply:
Good point, that part is OK then. I do find it a bit strange to use this only for some classes and not for others though.